lib/middleware/request-handler.ts

```diff
@@ -494,3 +494,36 @@ export function onUnsubscribe(req: Interface.IOurRequest,
             return next(new restify.InternalError(err.message));
         });
 }
+
+export function onChangeSubscriptions(req: Interface.IOurRequest,
+                                      res: Interface.IOurResponse,
+                                      next: restify.Next): void
+{
+    if (!req.apiSession) {
```

Comment:
This should probably be an assert, since this should never happen, unless something wrong with the `api-session` middleware.